students/laura-hernandez/ejercicio-1/tests/index.test.ts

```diff
@@ -0,0 +1,58 @@
+import  recursiveFibonacci from "../src/index";
+
+describe("Given a recursiveFibonacci function", () => {
+  describe("When it receives a 1", () => {
```

Comment:
Test cases description should be functional, not value coupled description. We will talk about it on Monday ;)